### PR TITLE
Githubmodule noconnection

### PR DESCRIPTION
**Pull request by Kokan**
This PR aims to fix the #810 and also contains small refactor for the github module.

### CODE REVIEW

**Comment by patrick96** [comment]
Thank you very much!
I'll have a look at this on the weekend.

Seems like maybe this is the root cause of #754

**Comment by Kokan** [comment]
Sure, I've added a log message with the exception.

**Comment by Kokan** [comment]
@patrick96 could you please check it ? thanks.

**Comment by patrick96** [comment]
Sorry, for the delay. I'll try to have a look tomorrow. If I don't, please remind me again in a few days.

File: src/modules/github.cpp [hunk 1]
```diff
@@ -25,21 +25,25 @@ namespace modules {
     if (m_formatter->has(TAG_LABEL)) {
       m_label = load_optional_label(m_conf, name(), TAG_LABEL, "Notifications: %notifications%");
 
-      if (m_empty_notifications) {
-        m_label->replace_token("%notifications%", "0");
-      } else {
-        m_label->clear();
-      }
+      update_label(-1);
```
Comment (patrick96) [inline src/modules/github.cpp]: I think we should keep the behaviour that the module is initialized with 0

Comment (Kokan) [reply src/modules/github.cpp]: Actually it is confusing for me to see no notification, when actually no information is present.
So I preferred to display other value then 0. 
I am planning to introduce the `label-offline` which would be the best option so I changed back to 0 for now :)

**Comment by patrick96** [comment]
Thank you very much!
Would really appreciate it, if you worked on the `label-offline` feature.

Btw. how are you committing your changes? The commit list only shows two commits but you have certainly more than that. Do you append old commits and force push?

**Comment by Kokan** [comment]
Yes, I used fixups for the commits, and force push. Github tracks those changes, and it is better for the history specially in case of rebase instead of merge :)

**Comment by patrick96** [comment]
Yes definitely helps keep the commit history clean. But it also makes it harder to see what was actually changed when something new is commited because you have to review the old commit again too.

**Pull request merged**